src/frame.rs

```diff
@@ -723,7 +723,7 @@ impl Frame {
 
             Frame::Datagram { data } => {
                 1 + // frame type
-                octets::varint_len(data.len() as u64) + // length
+                2 + // length, always encode as 2-byte varint
```

Comment:
I think this was correct before, as the length varint is not fixed length:
https://github.com/cloudflare/quiche/blob/master/src/frame.rs#L1148

Compare to e.g. https://github.com/cloudflare/quiche/blob/master/src/frame.rs#L1148

Reply:
(fixed the first link as I pasted the same thing twice...)

Reply:
yeah I  think you're correct